app/src/main/java/com/gholem/moneylab/features/planning/adapter/item/CoinCapItem.kt

```diff
@@ -1,6 +1,6 @@
 package com.gholem.moneylab.features.planning.adapter.item
 
-data class PersonsItem(
+data class CoinCapItem(
```

Comment:
1. Change the name to `ExchangeResponse`
2. Its a model for API and it's in planning feature adapter package - move it to proper package